src/App.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import './App.css';
-import Navbar from './components/Navbar/Navbar';
-import Carousel from './components/Carousel/Carousel';
+import Navbar from './components/organisms/Navbar/index';
```

Comment:
Usually no need `index` in the end for import just `import Navbar from './components/organisms/Navbar";` Please check for all similar places in your app.

Reply:
Fixed it in all files